app/errors.py

```diff
@@ -0,0 +1,14 @@
+class NotWearingMaskError(BaseException):
```

Comment:
Should add __init__() to initialize the class of exception and should be used Exception class as a parent, not a BaseException

Example:
class NotWearingMaskError(Exception):
def init(self, message):
super.init(message)
self.message = message